source/common/config/new_grpc_mux_impl.cc

```diff
@@ -51,6 +51,17 @@ bool NewGrpcMuxImpl::paused(const std::string& type_url) const {
   return pausable_ack_queue_.paused(type_url);
 }
 
+void NewGrpcMuxImpl::requestAliasesResolution(const std::string& type_url,
+                                              const std::set<std::string>& aliases) {
+  auto sub = subscriptions_.find(type_url);
+  RELEASE_ASSERT(
```

Comment:
Nit: this should be an `ASSERT` if it's an invariant, i.e. it's always going to be true, no need to enable in prod builds.